src/compiler/diagnosticMessages.json

```diff
@@ -5090,11 +5090,11 @@
         "category": "Error",
         "code": 6310
     },
-    "Project '{0}' is out of date because oldest output '{1}' is older than newest input '{2}'": {
+    "Project '{0}' is out of date because output '{1}' is older than input '{2}'": {

```

Comment:
Changed these messages to avoid using oldest and newest as that might not be true (Explained in tsbuild logic where we determine upto date ness check)